docs/manuals/source/TasksAndConcepts/PAM.rst

```diff
@@ -117,4 +117,4 @@ Users have limited access to commands and jobs. Therefore the appropriate rights
   }
 
 
-Additional information can be found at https://github.com/bareos/bareos-contrib/tree/master/misc/bareos_pam_integration
+Additional information can be found at https://github.com/bareos/bareos/tree/master/contrib/misc/bareos_pam_integration
```

Comment:
```suggestion
```
As User resources have changed with Bareos-19.2, this documentation will not work. Therefore it might be better to remove this sentence at all.